internal/federationout/federationout.go

```diff
@@ -298,20 +298,16 @@ func rawToken(ctx context.Context) (string, error) {
 
 func intersect(aa, bb []string) []string {
 	if len(aa) == 0 || len(bb) == 0 {
-		return []string{}
+		return nil
 	}
-	result := []string{}
```

Comment:
This is actually a behavior change. The result will be `nil` instead of `[]string{}` on not found.

Reply:
True, but they are almost indistinguishable. `nil` is a bit more idiomatic.